content/kubermatic/master/tutorials/12-audit-logs/_index.en.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Audit Logging"
+date = 2020-04-02T14:50:15+02:00
+weight = 120
++++
+
+Audit Logging is one of key security features provided by Kubernetes. Once enabled in kube-api, it provides a chronological record of operations performed on the cluster by users, administrators and other cluster components.
+
+Audit logging is also a key requirement of the [Kubernetes CIS benchmark](https://www.cisecurity.org/benchmark/kubernetes/).
+
+For more details, you can can refer to the [upstream documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/).
+
+### Kubermatic Support
+Kubermatic provideds two levels of support for the Audit Logging:
+
+* Audit Logging on user-cluster level
+* Audit Logging on a datacenter level
+
+{{% notice note %}}
+Kubernetes Audit Logging is optional and is not enabled by default, since it requires additional memory and storage resources, depending on the specific configuration used.
+{{% /notice %}}
+
+Once enabled, Kubermatic will use a [Log Backend](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#log-backend) and a minimal [Policy](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy) by default:
+
+```yaml
+{{< readfile "data/policy.yaml">}}
```

Comment:
I think this file is not there anymore. Please rebase the PR.

Reply:
@kron4eg, it is there. This PR adds it :+1:

Reply:
Please move it out from `/data` dir, in hugo `/data` is interpreted by hugo (see https://gohugo.io/templates/data-templates/ for more info).

The appropriate place for it will be `/kubermatic/master/data`

Reply:
@kron4eg Done. Thank you for clarifying!